src/metabase/driver/sparksql_databricks_v2.clj

```diff
@@ -20,15 +20,16 @@
             [metabase.driver.sql.query-processor :as sql.qp]
             [metabase.driver.sql.util :as sql.u]
             [metabase.driver.sql.util.unprepare :as unprepare]
-            [metabase.mbql.util :as mbql.u]
+            [metabase.legacy-mbql.util :as mbql.u]
+            [metabase.lib.metadata :as lib.metadata]
             [metabase.query-processor.store :as qp.store]
             [metabase.query-processor.util :as qputil]
             [metabase.query-processor.util.add-alias-info :as add]
-            [metabase.driver.sql.query-processor :as sql.qp]
+            [metabase.lib.metadata :as lib.metadata]
```

Comment:
Hi @shrodingers, honestly, I have no clue as to what these things are. What do you suggest?

Reply:
Ahah, just remove the line added (`[metabase.lib.metadata :as lib.metadata]`) , since it is already defined above (These are kind of imports, and the metabase.lib.metadata is imported just before)
```suggestion
```